tests/parsers/fixtures/pw/tot_magnetization/data-file.xml

```diff
@@ -0,0 +1,843 @@
+          1.000000000000000e0 0.000000000000000e0 0.000000000000000e0
+          0.000000000000000e0 0.000000000000000e0 -1.000000000000000e0
+          </rotation>
+      </symmetry>
+      <symmetry>
+        <info name="inv.  90 deg rotation - cart. axis [0,0,1]" class="not found">lattice_symmetry</info>
+        <rotation rank="2" dims="3 3" order="F">
+          0.000000000000000e0 1.000000000000000e0 0.000000000000000e0
+          -1.000000000000000e0 0.000000000000000e0 0.000000000000000e0
+          0.000000000000000e0 0.000000000000000e0 -1.000000000000000e0
+          </rotation>
+      </symmetry>
+      <symmetry>
+        <info name="inv. 180 deg rotation - cart. axis [1,0,1]" class="not found">lattice_symmetry</info>
+        <rotation rank="2" dims="3 3" order="F">
+          0.000000000000000e0 0.000000000000000e0 -1.000000000000000e0
+          0.000000000000000e0 1.000000000000000e0 0.000000000000000e0
+          -1.000000000000000e0 0.000000000000000e0 0.000000000000000e0
+          </rotation>
+      </symmetry>
+      <symmetry>
+        <info name="180 deg rotation - cart. axis [1,1,0]" class="not found">lattice_symmetry</info>
+        <rotation rank="2" dims="3 3" order="F">
+          0.000000000000000e0 1.000000000000000e0 0.000000000000000e0
+          1.000000000000000e0 0.000000000000000e0 0.000000000000000e0
+          0.000000000000000e0 0.000000000000000e0 -1.000000000000000e0
+          </rotation>
+      </symmetry>
+      <symmetry>
+        <info name="inv.  90 deg rotation - cart. axis [0,1,0]" class="not found">lattice_symmetry</info>
+        <rotation rank="2" dims="3 3" order="F">
+          0.000000000000000e0 0.000000000000000e0 -1.000000000000000e0
+          0.000000000000000e0 -1.000000000000000e0 0.000000000000000e0
+          1.000000000000000e0 0.000000000000000e0 0.000000000000000e0
+          </rotation>
+      </symmetry>
+      <symmetry>
+        <info name="inv.  90 deg rotation - cart. axis [0,-1,0]" class="not found">lattice_symmetry</info>
+        <rotation rank="2" dims="3 3" order="F">
+          0.000000000000000e0 0.000000000000000e0 1.000000000000000e0
+          0.000000000000000e0 -1.000000000000000e0 0.000000000000000e0
+          -1.000000000000000e0 0.000000000000000e0 0.000000000000000e0
+          </rotation>
+      </symmetry>
+      <symmetry>
+        <info name="inv. 180 deg rotation - cart. axis [0,1,1]" class="not found">lattice_symmetry</info>
+        <rotation rank="2" dims="3 3" order="F">
+          1.000000000000000e0 0.000000000000000e0 0.000000000000000e0
+          0.000000000000000e0 0.000000000000000e0 -1.000000000000000e0
+          0.000000000000000e0 -1.000000000000000e0 0.000000000000000e0
+          </rotation>
+      </symmetry>
+      <symmetry>
+        <info name="180 deg rotation - cart. axis [1,-1,0]" class="not found">lattice_symmetry</info>
+        <rotation rank="2" dims="3 3" order="F">
```

Comment:
You should be able to remove all symmetries and also all but one of the KS eigen states. This will save a lot of lines

Reply:
Note that I also had to adjust the `nks` element.